src/Compiler/Microsoft.AspNetCore.Razor.Language/test/IntegrationTests/ComponentCodeGenerationTestBase.cs

```diff
@@ -10494,18 +10495,24 @@ public void ProducesStandardLinePragmaForCSharpCode()
     [Parameter]
     public int IncrementAmount { get; set; }
 }
-", throwOnFailure: false);
+");
 
         // Assert
         AssertDocumentNodeMatchesBaseline(generated.CodeDocument);
         AssertCSharpDocumentMatchesBaseline(generated.CodeDocument);
-        CompileToAssembly(generated, throwOnFailure: false);
+        CompileToAssembly(generated,
```

Comment:
Looks like this *isn't* failing on CI. It seems weird that we would even expect it to, so possibly something up with your machine?

Reply:
It's not failing specifically only on windows. Both mac and linux fail with this error; this is something up with .NET Framework. I'll dig around.